### PR TITLE
Added DNF algorithm in C++

### PR DESCRIPTION
**Pull request by thundercipher**

### CODE REVIEW

**Comment by juhitiwari** [comment]
Hey @thundercipher Thanks for the addition! Can you update the README as well by adding the entry for DNF under C++ section?

**Comment by thundercipher** [comment]
Like, in a new pull request? Should I add another code along with it as
well?

**Comment by juhitiwari** [comment]
You can update the README for DNF under this PR itself.

File: README.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 We love pull requests from everyone. This repository is created for you to contribute different algorithms in different languages. There are separate folders for different languages.
 
+## Edit
```
Comment (juhitiwari) [inline README.md]: There is a section to add your algorithm in the readme. Please add it there, attaching screenshot for your reference. 
Screenshot from 2020-10-01 12-06-14

**Review by juhitiwari — CHANGES_REQUESTED** [review]
Please refer to the screenshot to understand where to add your algorithm

**Pull request merged**